src/sass/modules/buttons.scss

```diff
@@ -204,3 +204,26 @@
     color: $black-80;
   }
 }
+
+.button--checkbox {
```

Comment:
would it work to do:
```css
.button--checkbox {
  color: $black-80;
  &:hover...
  &:disabled...
 .fa-check-square...
}
```
without .fa-square?  I can't remember if the icon has to have the color on it entirely.. even if it does maybe `.fa-square` could be color inherit?